infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/brokerphases/BrokerEnvironmentFactory.java

```diff
@@ -199,6 +190,23 @@ private Container newContainer(
     return container;
   }
 
+  protected List<String> getCommandLineArgs(RuntimeIdentity runtimeId) {
+    return new ArrayList<String>(
+        Arrays.asList(
+            "-push-endpoint",
+            cheWebsocketEndpoint,
+            "-runtime-id",
+            String.format(
+                "%s:%s:%s",
+                runtimeId.getWorkspaceId(),
+                MoreObjects.firstNonNull(runtimeId.getEnvName(), ""),
+                runtimeId.getOwnerId()),
+            "-cacert",
+            certProvisioner.isConfigured() ? certProvisioner.getCertPath() : "",
+            "--registry-address",
+            Strings.nullToEmpty(pluginRegistryUrl)));
```

Comment:
This is unrelated to the current PR, but seeing `-argument` looks strange from a UNIX perspective, where usually we use two dashes for long args and one for the short version (e.g. `-v, --version`). Could we change this to 
```suggestion
            "--push-endpoint",
            cheWebsocketEndpoint,
            "--runtime-id",
            String.format(
                "%s:%s:%s",
                runtimeId.getWorkspaceId(),
                MoreObjects.firstNonNull(runtimeId.getEnvName(), ""),
                runtimeId.getOwnerId()),
            "--cacert",
            certProvisioner.isConfigured() ? certProvisioner.getCertPath() : "",
            "--registry-address",
            Strings.nullToEmpty(pluginRegistryUrl)));
```

Reply:
Make sense, will change the flags as it is possible to use both single and double dash.